docs/Way-of-Working/UbiquitousLanguage.md

```diff
@@ -0,0 +1,227 @@
+# Ubiquitous Language
+
+For clarity and consistency, this document defines key terms used within our team across various domains (e.g., Data
+Science, IT).
+
+The goal is to provide a list of commonly used terms and terms that are good to know. It is not to a full reference of
+all existing terms.
+
+Terms can and have been used 'as is' from sources mentioned at the end of this document.
+
+## Terms used in Data Science
+
+### Algorithm
+
+An algorithm is a sequence of repeatable steps, often expressed mathematically, written by a human and executed by a
+computer, to solve a certain type of data science problem. The main concept is that an algorithm takes in some input and
```

Comment:
```suggestion
An algorithm is a sequence of repeatable steps, often expressed mathematically, written by a human and executed by a
a human or computer, to solve a certain type of data science problem. The main concept is that an algorithm takes in some input and
```

Reply:
I would not limit the definition of an algorithm to "solve certain type of data science problems". The common definition of an algorithm is broader than this. I would take any definition from a standard textbook, for example the definition [here](https://ada-developers-academy.github.io/jumpstart-textbook/jumpstart-lessons/problem-solving/algorithms/#:~:text=In%20The%20Art%20of%20Computer,a%20reasonable%20amount%20of%20time) of Knuth coming from his classic textbook.